src/Surfnet/ServiceProviderDashboard/Infrastructure/Manage/Client/EntityChangeRequestClient.php

```diff
@@ -79,12 +80,12 @@ public function openChangeRequest(ManageEntity $entity, ?ManageEntity $pristineE
         return $response;
     }
 
-    public function getChangeRequest(string $id): array
+    public function getChangeRequest(string $id, Protocol $protocol): array
     {
         $this->logger->info(sprintf('Get outstanding change requests from manage for entity "%s"', $id));
 
         return $this->client->read(
-            '/manage/api/internal/change-requests/saml20_sp/' . $id
+            '/manage/api/internal/change-requests/' . $protocol->getManagedProtocol() . '/' . $id
```

Comment:
Now that we are starting to create a concoction of several strings, I feel it's time to use good 'ol `sprintf` to make things readable and maintainable.

Reply:
Agree